app/api/user_check_in.py

```diff
@@ -178,11 +203,14 @@ def before_create_object(self, data, _view_kwargs):
                         "Attendee already check daily on station.",
                     )
 
-        if station.station_type in (
-            STATION_TYPE.get('check in'),
-            STATION_TYPE.get('check out'),
-        ):
-            data['check_in_out_at'] = datetime.datetime.utcnow()
+        if station.station_type == STATION_TYPE.get('check in'):
+            attendee.is_checked_in = True
+            attendee.checkin_times = current_time
+            save_to_db(attendee)
+        if station.station_type == STATION_TYPE.get('check out'):
+            attendee.is_checked_out = True
+            attendee.checkout_times = current_time
+        save_to_db(attendee)
```

Comment:
Remove the save_to_db(attendee) at line 209 because you always save it to db.

Reply:
fixed